examples/LiteralObjects_Example.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
Let's have a brief discussion about the motivation behind this notebook when you go through the review.

Reply:
I think we can clean up the discussion a little.